Google.PowerShell/BigQuery/BqJob.cs

```diff
@@ -484,8 +607,41 @@ public Apis.Bigquery.v2.Data.Job DoCopy()
         /// </summary>
         public Apis.Bigquery.v2.Data.Job DoLoad()
         {
-            throw new NotImplementedException(
-                "Load jobs are not implemented yet.  Use Set-BqTabledata instead.");
+            if (ShouldProcess($"{Destination.TableId} (LOAD)"))
+            {
+                try
+                {
+                    var loadjob = new Apis.Bigquery.v2.Data.Job()
+                    {
+                        Configuration = new JobConfiguration()
+                        {
+                            Load = new JobConfigurationLoad()
+                            {
+                                DestinationTable = Destination,
+                                SourceFormat = (Type == DataFormats.JSON) ? "NEWLINE_DELIMITED_JSON" : Type.ToString(),
+                                SourceUris = SourceUris,
+                                Encoding = Encoding,
+                                FieldDelimiter = FieldDelimiter,
+                                Quote = Quote,
+                                MaxBadRecords = MaxBadRecords,
+                                SkipLeadingRows = SkipLeadingRows,
+                                IgnoreUnknownValues = AllowUnknownFields,
+                                AllowJaggedRows = AllowJaggedRows,
+                                AllowQuotedNewlines = AllowQuotedNewlines,
+                                WriteDisposition = WriteMode.ToString()
+                            }
+                        }
+                    };
+                    //deal with source format
+                    return Service.Jobs.Insert(loadjob, Project).Execute();
+                }
+                catch (Exception ex)
+                {
+                    ThrowTerminatingError(new ErrorRecord(ex, "Load Failed",
+                        ErrorCategory.InvalidOperation, this));
+                }
+            }
+            return null;
```

Comment:
That was for the compiler.  It's for the case that we Should (not) Process, and then neither the return or the throw will be hit.